tasks/promote.yml

```diff
@@ -1,63 +1,86 @@
 ---
```

Comment:
I don't really know this part of the code. @g0rgamesh should review.

Reply:
what do you suggest, if those versions are needed to get the ansible modules working?